geography_loader.py

```diff
@@ -0,0 +1,66 @@
+from nose.tools import set_trace
+import json
+from geoalchemy2 import Geography
+from sqlalchemy import func
+from sqlalchemy.sql.expression import cast
+
+from model import (
+    get_one_or_create,
+    Place,
+)
+
+class GeographyLoader(object):
+    """Load Place objects from a NDJSON document like that generated by
+    geojson-places-us.
+    """
+
+    def __init__(self, _db):
+        self._db = _db
+        self.places_by_external_id=dict()
+        
+    def load_ndjson(self, fh):
+        while True:
+            metadata = fh.readline().strip()
+            if not metadata:
+                # End of file.
+                break
+            geography = fh.readline().strip()
+            yield self.load(metadata, geography)
+
+    def load(self, metadata, geography):
+        metadata = json.loads(metadata)
+        external_id = metadata['id']
+        type = metadata['type']
+        parent_external_id = metadata['parent_id']
+        name = metadata['name']
+        aliases = metadata.get('aliases', [])
+        abbreviated_name = metadata.get('abbreviated_name', None)
+
+        if parent_external_id:
+            parent = self.places_by_external_id[parent_external_id]
```

Comment:
Yes, I do the US first, then all the states, and everything else (counties, cities, ZIP) is underneath the states.